beets/ui/__init__.py

```diff
@@ -99,6 +99,12 @@ def print_(*strings):
     replaces it.
     """
     if strings:
+        if not isinstance(strings[0], basestring):
+            try:
+                strings = map(unicode, strings)
+            except UnicodeError:
+                strings = map(bytes, strings)
```

Comment:
This type dependence is a little worrisome to me. It seems rather easy to make a mistake where you encode the wrong chunk just because you used a non-string in the first position. Can we just ask callers of `print_` to explicitly use `str(o)` or `unicode(o)` to sidestep this issue?